apps/parsley/package.json

```diff
@@ -112,28 +110,14 @@
     "@types/prompts": "2.4.4",
     "@types/react": "18.0.15",
     "@types/react-dom": "18.2.25",
-    "@typescript-eslint/eslint-plugin": "6.19.1",
-    "@typescript-eslint/parser": "5.62.0",
     "@vitejs/plugin-react": "3.0.1",
     "babel-plugin-import-graphql": "2.8.1",
     "cypress": "12.7.0",
     "cypress-real-events": "1.11.0",
     "env-cmd": "10.1.0",
     "eslint": "8.31.0",
```

Comment:
can this eslint be deleted? I see it in `eslint-config-evg` but I'm not sure. Also for prettier, typescript and graphql

Reply:
This `eslint` is required since it's used by the eslint Yarn scripts for each app

Reply:
I tried running a [patch](https://spruce.mongodb.com/version/662c1b6c6282860007d799dd/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) after removing eslint in each package.json and I think it worked, but maybe I'm looking at the wrong thing. 😅 But also maybe another acceptable thing would be to put eslint in the parent package.json and use `yarn cwd`

Reply:
thank you for the explanation!!